### ISSUE TITLE
Readme getting outdated

### ISSUE THREAD

**Issue by nem0** [Alan-FGR/aelum#18]
as of now, it has not been updated for 3 hours, it's getting obsolete.

**Comment by Alan-FGR** [comment]
I agree, we will be hiring people working around the clock to keep the readme always fresh!
